### PR TITLE
Readme spelling

### PR DESCRIPTION
**Pull request by timjmartel**
One spelling and on capitalization issue found during adoption in readme.md

### CODE REVIEW

File: react-sdk.code-workspace [hunk 1]
```diff
@@ -2,11 +2,14 @@
 	"folders": [
 		{
 			"path": "."
+		},
+		{
+			"path": "../../scripts"
```
Comment (JEAxero) [inline react-sdk.code-workspace:7]: I don't think we can assume that everyone has a scripts directory 2 levels up. (I know I don't). And I think this will cause problems. So, I'd revert this change with another commit to put it back the way it was. Then, this PR looks ok.

**Review by JEAxero — CHANGES_REQUESTED** [review]

File: react-sdk.code-workspace (hunk 1 above)
Comment (timjmartel) [reply react-sdk.code-workspace:7]: ok, tried original commit from VSCode.  Staged more than just the readme.md

Comment (JEAxero) [reply react-sdk.code-workspace:7]: Thanks!

**Review by JEAxero — APPROVED** [review]
Looks good now. Thanks!
